test/integration/export/test/ssr.js

```diff
@@ -14,7 +14,7 @@ export default function (context) {
       const $ = cheerio.load(html)
       const dynamicLink = $('#dynamic-1').prop('href')
       const filePathLink = $('#path-with-extension').prop('href')
-      expect(dynamicLink).toEqual('/dynamic/one/')
+      expect(dynamicLink).toEqual('/dynamic/one')
```

Comment:
This changed because of:

```
   // We need this for server rendering the Link component.
   global.__NEXT_DATA__ = {
     nextExport: true
   }
```

Not being set in the worker. If you add it there it'll work.

We can remove this hack in the future when we add `next build --target=export`

Reply:
Ah, good to know! I've added the global var and reset the test.